webui/src/App.js

```diff
@@ -409,9 +409,12 @@ function ValueMonitors(props) {
 
 function Plot(props) {
   const canvasRef = React.useRef(null);
-  const colors = ['red', 'orange', 'green', 'blue'];
-  const display = [true, true, true, true];
+  const numSensors = props.numSensors;
   const webUIDataRef = props.webUIDataRef;
+  const display = new Array(numSensors).fill(true);
+  const colors = ['red', 'orange', 'green', 'blue'];
```

Comment:
Instead of cycling through colors do you think we can use hsl values? This way the colors for every sensor would be somewhat distinct.

e.g. for 4 sensors the 4 values we will use are the following (360/#sensors = increment value):
`hsl(0, 100%, 50%)`
`hsl(90, 100%, 50%)`
`hsl(180, 100%, 50%)`
`hsl(270, 100%, 50%)`

And then below where we set the dark variant we instead just set the last argument to 30% or so
`hsl(0, 100%, 30%)`
`hsl(90, 100%, 30%)`
`hsl(180, 100%, 30%)`
`hsl(270, 100%, 30%)`

Reply:
> Instead of cycling through colors do you think we can use hsl values?

Oh yes I was going to comment about this but I forgot. I think cycling makes sense to not put an arbitrary limit on the number of sensors, but there should be more than four colors here. Mostly, I wasn't sure what colors to pick.

Reply:
Ah, now I get it! Nice. Thanks for the code example.